### DISCUSSION TITLE
[Prettier Plugin] not working at all in next-js 13.2 .

### DISCUSSION THREAD

**Discussion by vinaykulk621** [tailwindlabs/tailwindcss#10710] [Help] [answered]
I am using next js 13.2 configured tailwind properly, it is working as it needs to.
The sort functionality of prettier plugin not working at all. i followed this [tutorial-from-official-tailwindcss](https://www.youtube.com/watch?v=_CntOc4hBcg&ab_channel=TailwindLabs),
I did exactly as he mentioned, but it is not working for me

prettier extension   -->9.10.4
tailwindcss              -->3.2.4

`package.json`
```json
  "devDependencies": {
    "autoprefixer": "^10.4.13",
    "postcss": "^8.4.21",
    "prettier": "^2.8.4",
    "prettier-plugin-tailwindcss": "^0.2.3",
    "tailwindcss": "^3.2.4",
}
```

`.prettierrc.json`
```json
{
  "tabWidth": 2,
  "trailingComma": "es5",
  "semi": false
}
```

`tailwind.config.js`
```js
/** @type {import('tailwindcss').Config} */
module.exports = {
  content: [
    "./app/**/*.{js,ts,jsx,tsx}",
    "./components/**/*.{js,ts,jsx,tsx}"
  ],
  theme: {
    extend: {
      colors: {
        "simple-black": "#0c0c0c",
        "simple-black-2": "#171717",
        "simple-blue": "#0a2725",
      },
      fontFamily: {
        sans: ['var(--font-Inter)'],
        alerta: ['var(--font-Alerta)'],
        spotify: ['spotify-circular', 'spotify-circular-cyrillic', 'spotify-circular-arabic', 'spotify-circular-hebrew', 'Helvetica Neue', 'helvetica', 'arial', 'Hiragino Kaku Gothic Pro', 'Meiryo', 'MS Gothic']
      }
    },
  },
}```

**Comment by luizov** [comment, marked as answer]
You have added the plugin as a dependency but haven't added it in the prettier configuration file. 
In your `.prettierrc.json` add the following:
`"plugins": ["prettier-plugin-tailwindcss"]`

Your `.prettierrc.json` should now look like this:
```
{
  "plugins": ["prettier-plugin-tailwindcss"],
  "tabWidth": 2,
  "trailingComma": "es5",
  "semi": false
}
```

**Reply by vinaykulk621** [reply]
Thanks

**Reply by codedrian** [reply]
Hi. Where did you find the `.prettierrc.json`.  [Here](https://github.com/Codedrian-Code-Studio/tailwind-craft-station.git) is my project repository and I can't find the `.prettierrc.json`. @vinaykulk621 @luizov 
[Here's](https://github.com/tailwindlabs/tailwindcss/discussions/10717) my problem. Thanks.

**Reply by luizov** [reply]
@codedrian You create it manually in your project folder. Add new file -> name it .prettierrc.json in the root directory and configure your settings inside.

**Reply by codedrian** [reply]
I just realize I didn't set the "prettier.prettierPath" option in my VS Code settings to the path of the Prettier binary installed in my project's node_modules folder `"prettier.prettierPath": "./node_modules/prettier/bin-prettier.js"`. Is the path I set correct? I saw some comment in stackoverflow that I should set it to `.prettierrc`.

**Reply by luizov** [reply]
That is only needed when you install prettier globally in your system, this is not needed when you install prettier locally in your project using `npm install --save-dev prettier`. Please let's not flood this topic though as the original question is related to `prettier-plugin-tailwindcss`. Thank you.
